src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+import './Banner.css';
+
+interface IProps {
+  message: string;
+  tooltipMessages: string[];
+}
+
+const Banner = (props: IProps) => {
+  const [dismissed, setDismissed] = useState(false);
+  const [tooltipHidden, setTooltipHidden] = useState(true);
+
+  return (
+    <div className={`banner ${dismissed && 'dismissed'}`}>
+      <div
+        onMouseEnter={() => setTooltipHidden(false)}
+        onMouseLeave={() => setTooltipHidden(true)}
```

Comment:
Would a `toggleTooltip` function work better here? As that will only require creating a single handler function to eliminate the need for anon functions